PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetPerformance.cxx

```diff
@@ -2190,7 +2228,30 @@ void AliAnalysisTaskEmcalJetPerformance::ComputeBackground()
   AliJetContainer* jetCont = 0;
   TIter nextJetColl(&fJetCollArray);
   while ((jetCont = static_cast<AliJetContainer*>(nextJetColl()))) {
-    
+	  if(jetCont->GetNJets()<1)continue;//Don't enter if there are no jets in the container
```

Comment:
Can you please add a comment describing what this section of code is doing? (The whole block here, I mean)

Reply:
the line that I have added has a comment attached to it. I didn't change anything else